examples/server/deps.sh

```diff
@@ -5,14 +5,24 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 PUBLIC=$DIR/public
 
-echo "download js bundle files"
-curl https://npm.reversehttp.com/@preact/signals-core,@preact/signals,htm/preact,preact,preact/hooks > $PUBLIC/index.js
-echo >> $PUBLIC/index.js # add newline
+if [ "$1" != "--no-download" ]; then
+  echo "download js bundle files"
+  curl https://npm.reversehttp.com/@preact/signals-core,@preact/signals,htm/preact,preact,preact/hooks > $PUBLIC/index.js
+  echo >> $PUBLIC/index.js # add newline
+fi
 
 FILES=$(ls $PUBLIC)
+UNAME_S=$(uname -s)
 
 for FILE in $FILES; do
   func=$(echo $FILE | tr '.' '_')
   echo "generate $FILE.hpp ($func)"
-  xxd -n $func -i $PUBLIC/$FILE > $DIR/$FILE.hpp
```

Comment:
It would be simpler to just `cd $PUBLIC` at the beginning of the script instead of having to post-process the output of old versions of xxd.